### PR TITLE
Homework/homework 1

### PR DESCRIPTION
**Pull request by VictorKozhevnikov**
Hi. Here's the pull request for updated homework-1 branch

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,15 +4,23 @@
   "description": "nodejs tutorial project",
   "main": "index.js",
   "scripts": {
-    "start": "node ./main.js",
-    "test": "node ./test.js"
+    "start": "node ./dist/main.bundle.js",
+    "test": "node ./test.js",
+    "build": "npm run webpack",
```
Comment (ubcent) [inline package.json]: Do you realy need webpack for compilation? Maybe it's better to use ts cli? See [tsc](https://www.typescriptlang.org/docs/handbook/compiler-options.html)

File: src/index.ts [hunk 2]
```diff
@@ -0,0 +1,10 @@
+// have to use require because ts compiler can not import this
+const config = require('./config');
```
Comment (ubcent) [inline src/index.ts]: You should change config file type. json -> ts

File: package.json [hunk 3]
```diff
@@ -4,15 +4,23 @@
...
+    "build-watch": "npm run webpack -- --watch",
```
Comment (ubcent) [inline package.json]: I recommend a nodemon for watching instead of webpack

**Comment by VictorKozhevnikov** [comment]
Hi. I have used tcs for build. It's easy :)

**Pull request merged**